.circleci/config.yml

```diff
@@ -167,7 +191,8 @@ jobs:
       - store_artifacts:
           path: ./build/coverage/acceptance.html
           destination: acceptance-coverage.html
```

Comment:
Just the fact we spin up a new environment with local dependencies such as Besu, Kafka, Hashicorp etc it is relevant here